packages/sourcecred/src/plugins/github/relationalView.js

```diff
@@ -47,7 +47,7 @@ export class RelationalView {
   _mapReferences: Map<N.RawAddress, N.ReferentAddress[]>;
   _mapReferencedBy: Map<N.RawAddress, N.TextContentAddress[]>;
 
-  constructor(): void {
+  constructor(repository?: T.Repository): void {
     this._repos = new Map();
```

Comment:
Technically we do not need  `_repos` map as now only one repository, but moving to singular repository field  would require more significant changes (since `RelationalView` API will be altered). Maybe worthwhile to create a follow-up task to clean this up?